swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/services/BillableUsageService.java

```diff
@@ -52,10 +55,15 @@ public class BillableUsageService {
 
   private static final ContractCoverage DEFAULT_CONTRACT_COVERAGE =
       ContractCoverage.builder().total(0).gratis(false).build();
+  protected static final String COVERED_USAGE_METRIC =
```

Comment:
According to the google document linked to the epic, the metric should be only "swatch_contract_usage_total".